srcpkgs/QTAlarm/template

```diff
@@ -0,0 +1,24 @@
+# Template file for 'QTAlarm'
+pkgname=QTAlarm
+version=2.3.2
+revision=1
+build_style=qmake
+hostmakedepends="qt5-qmake qt5-host-tools"
+makedepends="qt5-multimedia-devel qt5-declarative-devel"
+short_desc="Alarm clock written in QT"
+maintainer="Charles Haines <charlesmhainesii@gmail.com>"
+license="GPL-3.0-only"
+homepage="https://random-hackery.net/page/qtalarm/"
+distfiles="https://github.com/CountMurphy/QTalarm/archive/refs/tags/${version}.tar.gz"
+checksum="bc6148366e0639e58190719f68bbbb261b618c5e0f51ccf16cc54c5266c604d0"
+
+do_install() {
+	cd ${wrksrc}
```

Comment:
Is this necessary?

Reply:
Yes, I think the `cd ${wrksrc}` line can be removed.

Reply:
cd ${wrksrc} removed.

Reply:
Please also remove `srcpkgs/QTAlarm/qtalarm.desktop`.

Reply:
Can it be included if it's in the files subdirectory or must I remove it completely?

Reply:
Currently it's in both but it should only be in the files subdirectory.

Reply:
Ok thanks. I'll get rid of the duplicate.